src/app/zap-templates/zcl/data-model/chip/camera-av-settings-user-level-management-cluster.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/CameraAVSettingsUserLevelManagement.adoc
+Parameters: in-progress
+Git: 1.4-267-gc3d95b5c9
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="Cameras"/>
+  <struct name="MPTZPresetStruct" apiMaturity="provisional">
```

Comment:
I really wish Alchemy output things in spec order, not reorder them....  Would make it much easier to review.